server/src/test/java/org/apache/druid/client/JsonParserIteratorTest.java

```diff
@@ -316,8 +316,41 @@ private Query<?> mockQuery(String queryId, long timeoutAt)
     }
   }
 
+  public static class IAEExceptionConversionTest
+  {
+    @Rule
+    public ExpectedException expectedException = ExpectedException.none();
+
+    private String meshErrMsg = "pstream connect error or disconnect/reset before header";
```

Comment:
Also, can we add a test case with null value. 
```suggestion
    private String errorMessage = "pstream connect error or disconnect/reset before header";
```

Reply:
added the null test case as suggested.